playbooks/common/openshift-node/config.yml

```diff
@@ -27,10 +27,12 @@
     stat:
       path: "{{ item }}"
     with_items:
-    - "/etc/openshift/node/node.key"
-    - "/etc/openshift/node/node.kubeconfig"
+    - "/etc/openshift/node/system:node:{{ openshift.common.hostname }}.crt"
+    - "/etc/openshift/node/system:node:{{ openshift.common.hostname }}.key"
+    - "/etc/openshift/node/system:node:{{ openshift.common.hostname }}.kubeconfig"
```

Comment:
@liggitt @sdodson There is no real reason that we couldn't use create-node-config, the only reason I didn't was to make it explicitly clear that we are not using the generated node config, and just using the generated server certs and generated client certs/kubeconfig.  We could easily go back to create-node-config and throw away the generated node config and node json files (preferably before we create the tarball and transfer over to the node host).